clients/admin-ui/src/features/integrations/configure-monitor/ConfigureWebsiteMonitorForm.tsx

```diff
@@ -114,6 +114,14 @@ const ConfigureWebsiteMonitorForm = ({
     onSubmit(payload);
   };
 
+  // Website monitors shouldn't support daily or weekly frequencies
+  const frequencyOptions = enumToOptions(MonitorFrequency).filter(
+    (option) =>
+      ![MonitorFrequency.DAILY, MonitorFrequency.WEEKLY].includes(
+        option.value as MonitorFrequency,
+      ),
+  );
+
```

Comment:
it seems like it could be slightly clearer/more maintainable to define this as its own enum that 'extends' the `MonitorFrequency` enum, but i couldn't figure out a nice way to make that work...

Reply:
in https://github.com/ethyca/fides/pull/5981/commits/7cc7c535bfa3ed30ae0f167f7c55454cce619e98 i updated this to enumerate the frequency values we _do_ want, specifically - this feels a bit less error-prone moving forward, since if any of those frequencies are removed from the underlying `MonitorFrequency` enum, then we'll get typing errors; it will also ensure that we don't accidentally sneak "new" frequencies into the web monitor form. we'd rather be restrictive here, IMO - so i think the inclusive approach is a bit safer as we move forward.

that being said, a new 'proper' enum that somehow extends the underlying enum would be nicest, just not sure that's something that can be implemented reasonably.